e2e/helpers/browserback_helper.js

```diff
@@ -6,4 +6,9 @@ module.exports = class MyHelpers extends Helper {
     const page = this.helpers['Puppeteer'].page;
     return page.goBack();
   }
+
+  refreshThePage() {
```

Comment:
Why did you have to implement custom helper for page refresh?

Reply:
Hi @geek-soft ,
When I use the codeceptjs direct method I.refreshPage() then it is exiting the shell and the control is not returning it, hence I used custom helper for the page refresh.